docs/compose.md

```diff
@@ -67,11 +67,33 @@ FROM nvcr.io/nvidia/tritonserver:<xx.yy>-py3-min
 COPY --from=full /opt/tritonserver/bin /opt/tritonserver/bin
 COPY --from=full /opt/tritonserver/lib /opt/tritonserver/lib
 ```
-
+Then install dependencies outlined in [build.py](https://github.com/triton-inference-server/server/blob/main/build.py), `dockerfile_add_installation_linux()` function.
+ 
+Example Dockerfile that only uses `Tensorflow 1` with the 21.06 build:
+```
+FROM nvcr.io/nvidia/tritonserver:21.06-py3 as full
+FROM nvcr.io/nvidia/tritonserver:21.06-py3-min
+COPY --from=full /opt/tritonserver/bin /opt/tritonserver/bin
```

Comment:
I don't like this because it is not complete. I think we should stop documenting the manual way of doing compose and instead just change this documentation to only refer to the compose.py script and how to use it.